### ISSUE TITLE
Field title with a dot

### ISSUE THREAD

**Issue by pbuccigrossi** [react-csv/react-csv#153]
The columns with a dot in field title are not popluated in csv file. I notice this behaviour on new version 1.1.1, until 1.0.12 this bug was not present.

**Comment by mccabemj** [comment]
@pbuccigrossi yep I can replicate. Will take a look when I get a chance unless someone can help

**Comment by mccabemj** [comment]
@pbuccigrossi this is related to the change I made to support dot notation in headers

https://github.com/react-csv/react-csv/commit/7cade4747a2f30e3af6c1d54bf7dc513d04e4d07

It is splitting based on the dot and then not finding anything that matches. Will need to come up with a way to support both.

In the meantime, you can define your own headers, and only put the dots in the `label` not the `key`. This may obviously mean you need to map your data to be called different too.  But it would at least allow you to have dots in your headers

```
  const headers = [
    { label: 'First Name.', key: 'firstname' },
    { label: 'Last Name.', key: 'lastname' },
    { label: 'E.m.a.i.l.', key: 'email' }
  ];

  const csvData = [
    { firstname: 'Ahmed', lastname: 'Tomi', email: 'ah@smthing.co.com' },
    { firstname: 'Raed', lastname: 'Labes', email: 'rl@smthing.co.com' },
    { firstname: 'Yezzi', lastname: 'Min l3b', email: 'ymin@cocococo.com' }
  ];
```

**Comment by hmudimi** [comment]
@mccabemj If you want I can help you with this bug, Do you still need help?

**Comment by mccabemj** [comment]
@hmudimi yeah, any help would be appreciated.

**Comment by hmudimi** [comment]
@mccabemj I have tried with sample data 
```
const csvHeaders = [
"Com.pany","جهة الإتصال ","王玉.普"
]
const csvData =[
  ['Alfreds Futterkiste'	,'Maria Anders',	'Germany'] ,
  ['Rathath IT', 'Abdennour TM' , 'تونس'] ,
  ['Sinopec', '王玉普' , '中国'],
  ['Auto1', 'Petter' , 'Germany'] ,
  ['Estifeda', 'Yousri K' , 'تونس'] ,
  ['Nine 10ᵗʰ', 'Amjed Idris' , 'المملكة العربية السعودية '] ,
  ['Tamkeen', 'Mohamed Alshibi' , 'المملكة العربية السعودية'] ,
  ['Packet Publishing', 'David Become' , 'UK'] ,
  ['Soft.ware hourse', 'So.ro' , 'Poland']
];
```
 and
```
const csvHeaders = [
  { label: 'First. Name', key: 'first.name' },
  { label: 'Last Name.', key: 'lastname' },
  { label: 'E.m.a.i.l.', key: 'email' }
];

const csvData = [
  { first:{name: 'Ahm.ed'}, lastname: 'Tomi', email: 'ah@smthing.co.com' },
  { first:{name: 'Raed'}, lastname: 'Lab.es', email: 'rl@smthing.co.com' },
  { first:{name: 'Yezzi'}, lastname: 'Min l3b', email: 'ymin@cocococo.com' }
];
```

but I am not able to reproduce this issue with the master branch if you help me to reproduce it.

**Comment by hmudimi** [comment]
@mccabemj I have created a PR for this bug.

**Comment by mccabemj** [comment]
Fixed with PR #169 thanks @hmudimi for your help
